crates/runtime/src/embeddings/vector_search.rs

```diff
@@ -110,10 +112,19 @@ impl VectorSearch {
         embedding_column: &str,
         n: usize,
     ) -> Result<(Vec<std::string::String>, Vec<arrow::array::RecordBatch>)> {
+        // Need to handle quoting projection columns manually (when needed).
```

Comment:
Bug occurs when there are primary keys that would have needed quoting. 
For example 
```sql
SELECT cveId, description_embedding FROM...
```

Needs to be 
```sql
SELECT "cveId", desciption_embedding FROM...
```